src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/FenghourCO2Viscosity.hpp

```diff
@@ -75,7 +75,8 @@ class FenghourCO2Viscosity : public PVTFunctionBase
                         string_array const & inputParams,
                         string_array const & componentNames,
                         array1d< real64 > const & componentMolarWeight,
-                        bool const printTable );
+                        bool const printInCsv,
+                        bool const printInLog );
```

Comment:
Should this be a `PVTOutputOptions` struct ?

Reply:
Can't declare a struct without including header everywhere

Reply:
I would be surprised that something like a `struct PVTFunctionBase::PVTOutputOptions` would not work.

Reply:
Name could be also `struct PVTFunctionBase::TableOutputOptions`.

Reply:
it's done  now !